draft-spaghetti-sidrops-rpki-ta-tiebreaker.xml

```diff
@@ -88,14 +88,15 @@ version="3">
         TA operators periodically reissue TA certificates to introduce changes to, for example, modify the set of IP Address Delegations and/or Autonomous System Identifier Delegations included in the <xref target="RFC3779"/> extension(s), the content of the Subject Information Access extension (<xref target="RFC5280" section="4.2.2.2"/>), and the certificate validity period (<xref target="RFC5280" section="4.1.2.5"/>).
       </t>
       <t>
-        Relying Parties periodically download purported current Trust Anchor certificates from remote locations and verify that the key of the self-signed certificate matches the key embedded in its associated Trust Anchor Locator <xref target="RFC8630"/>.
-        This means that Relying Parties have a choice between using a previously retrieved locally cached copy of the TA certificate and the newly-retrieved instance of the TA certificate.
+        Relying Parties periodically download Trust Anchor certificates from well-known, remote locations and verify that the key of the self-signed certificate matches the key embedded in its associated Trust Anchor Locator <xref target="RFC8630"/>.
+        This transfer may happen via an unauthenticated channel, and the certificate is verified by checking that it is signed by the public key in the TAL.
+        After retrieving a TA certificate Relying Parties have a choice between using a previously retrieved locally cached copy of the TA certificate and the newly-retrieved instance of the TA certificate.
       </t>
       <t>
-        In the past some Trust Anchor certificates were issued with egregious validity periods, in some cases up to a century.
-        Trust Anchor operators cannot revoke previously issued instances of TA certificates.
-        This means that an on-path adversary or caching network elements could present Relying Parties with an older instance of the TA certificate than the TA operator intended for Relying Parties to use.
-        This document proposes a tiebreaking scheme for Relying Parties to prefer the 'more recently' issued instance of the TA certificate using the notBefore date as an indicator and as second tiebreaker to prefer TA certificates with a shorter validity window. 
+        Some Trust Anchor certificates were issued with extended validity periods, in some cases up to a century.
```

Comment:
It feels very long, but I can not think of an attack that would be possible with a 100y certificate, but not with a 10y certificate. Manifests are what constrains validity.

(To me extended validity and the replay-attacks it introduces are a risk)

Reply:
@botovq if you can think of an attack, please mention it. It would make the case stronger.

Reply:
I think we want to communicate that some issuances have been for "unreasonably" or "unnecessarily" long validity periods.

If TA operators periodically reissue, it forces RPs to periodically refetch, which together makes it possible for TA operators to have some agility (agility which for example can be used to update the SIA rpkiManifest location). I know that already today all validators use TALs, not the actual TA certificates themselves, and "not-extended" validity periods reinforce that.

Parameters that can change in TA certificates:

A) validity period
B) INR set
C) SIA

Item (A) is the only one that can be used as an instrument to get change going for items (B) and (C). For both (B) and (C) 100 years is entirely unreasonable, because we can reasonably expect both (B) and (C) to change every few years. Both (B) and (C) *have* changed for some TA certs in the last few years. LACNIC recently updated their SIA, and all RIRs went from "exact INRs" to "0.0.0.0/0 + * INRs".

Using the length of the validity period as a tiebreaker and preferring shorter rather than longer validity windows helps both TA & RP side of the house remain agile for items (B) and (C).

I'd suggest to replace "egregious"/"extended" with "unreasonable", and if that word doesn't resonate well the backup suggestion is "concerningly"